### PR TITLE
fix(dashboard, types): ability to update price list status on admin

### PR DESCRIPTION
**Pull request by riqwan**
what:

- adds ability to update price list status on admin

price-list

RESOLVES CORE-2258

### CODE REVIEW

File: packages/admin-next/dashboard/src/routes/pricing/common/utils.ts [hunk 1]
```diff
@@ -25,25 +25,30 @@ export const getPriceListStatus = (
...
-  if (isScheduled) {
-    color = "orange"
-    text = t("pricing.status.scheduled")
-  }
+  let status: string = PriceListStatus.ACTIVE
 
   if (isDraft) {
     color = "grey"
     text = t("pricing.status.draft")
+    status = PriceListStatus.DRAFT
```
Comment (sradevski) [inline packages/admin-next/dashboard/src/routes/pricing/common/utils.ts:33]: Why are we using 2 different enums here for draft and the other two states?

**Review by shahednasser — APPROVED** [review]
LGTM works as expected

File: packages/admin-next/dashboard/src/routes/pricing/common/utils.ts (hunk 1 above)
Comment (riqwan) [reply packages/admin-next/dashboard/src/routes/pricing/common/utils.ts:33]: One is actual db status, the other is computed status that we display on the UI based on the date

Comment (sradevski) [reply packages/admin-next/dashboard/src/routes/pricing/common/utils.ts:33]: @riqwan I see, ideally the UI then defines all statuses, basically a union of the computed + db in one enum

Comment (riqwan) [reply packages/admin-next/dashboard/src/routes/pricing/common/utils.ts:33]: Hm, I don't know if that adds much value here for a simple badge. If we use it in other places, sure but that doesn't seem to be the case yet.

**Pull request merged**